woningwaardering/stelsels/utils.py

```diff
@@ -1,5 +1,6 @@
 from datetime import date, datetime, time
 from decimal import ROUND_HALF_UP, Decimal
```

Comment:
Deze import is nodig voor een typing fix verderop

```suggestion
from decimal import ROUND_HALF_UP, Decimal
from typing import Callable, List, Tuple
```